hello_world.txt

```diff
@@ -0,0 +1 @@
+Hello, World!
```

Comment:
This seems like an error ? Especially at the repo root ? I can't seem to find a test that would be using this or anything.

Reply:
Oops, totally an error of course. Sorry 🙈 I'll remove it.

Reply:
Removed. Thanks for the eagle eyes!